usaspending_api/database_scripts/matview_generator/universal_transaction_matview.json

```diff
@@ -71,7 +71,6 @@
     "  transaction_fabs.cfda_number,",
     "  references_cfda.program_title AS cfda_title,",
     "",
-    "  transaction_normalized.recipient_id,",
```

Comment:
Same for this matview: usaspending_api/awards/models_matviews/universal_transaction_matview.py